cmd/geth/main.go

```diff
@@ -325,3 +319,31 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 		}
 	}
 }
+
+// unlockAccounts unlocks any account specifically requested.
+func unlockAccounts(ctx *cli.Context, stack *node.Node) {
+	// personalExposed checks whether personal API is exposed by http.
+	personalExposed := func() bool {
```

Comment:
why not just write this as a function?

Reply:
I think this function is specialized and may not be invoked by other part codebase. So maybe embedded function is fine here.